docs/Two-Pointers/Max_Distance.md

```diff
@@ -0,0 +1,81 @@
+---
+id: practice-problems-on-two-pointers
+title: Practice Problems
+sidebar_label: Practice Problems
+sidebar_position: 3
+description: Here are some practice problems for the Two Pointers technique, including an implementation to find the maximum distance under the constraint arr[i] <= arr[j].
```

Comment:
replace `description: Here are some practice problems for the Two Pointers technique, including an implementation to find the maximum distance under the constraint arr[i] <= arr[j].` with `description: "Here are some practice problems for the Two Pointers technique, including an implementation to find the maximum distance under the constraint arr[i] <= arr[j]."` and my suggestion for you can't use `arr[i] <= arr[j].` in `description` because this is not good practice

Reply:
okay i will update it